Avoid build error in CLANGARM64 environment:
```
  [669/1237] Compiling C++ object sys/d3d11/libgstd3d11.dll.p/gstd3d11window_swapchainpanel.cpp.obj
  FAILED: sys/d3d11/libgstd3d11.dll.p/gstd3d11window_swapchainpanel.cpp.obj 
  "clang++" "-Isys/d3d11/libgstd3d11.dll.p" "-Isys/d3d11" "-I../gst-plugins-bad-1.24.9/sys/d3d11" "-I." "-I../gst-plugins-bad-1.24.9" "-Igst-libs" "-I../gst-plugins-bad-1.24.9/gst-libs" "-Igst-libs/gst/codecs" "-Igst-libs/gst/d3d11" "-Igst-libs/gst/dxva" "-IC:/msys64/clangarm64/include/gstreamer-1.0" "-IC:/msys64/clangarm64/include/glib-2.0" "-IC:/msys64/clangarm64/lib/glib-2.0/include" "-IC:/msys64/clangarm64/include/orc-0.4" "-IC:/msys64/clangarm64/include/directxmath" "-fdiagnostics-color=always" "-D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_FAST" "-D_FILE_OFFSET_BITS=64" "-Wall" "-Winvalid-pch" "-Wno-non-virtual-dtor" "-fvisibility=hidden" "-fno-strict-aliasing" "-Wformat-nonliteral" "-Wmissing-declarations" "-Wredundant-decls" "-Wwrite-strings" "-Wformat" "-Wformat-security" "-Winit-self" "-Wmissing-include-dirs" "-Waddress" "-Wno-multichar" "-Wvla" "-Wpointer-arith" "-O2" "-pipe" "-Wp,-D_FORTIFY_SOURCE=2" "-fstack-protector-strong" "-Wp,-D__USE_MINGW_ANSI_STDIO=1" "-DHAVE_CONFIG_H" "-DGST_USE_UNSTABLE_API" "-DHAVE_WINMM" "-Wno-redundant-decls" -MD -MQ sys/d3d11/libgstd3d11.dll.p/gstd3d11window_swapchainpanel.cpp.obj -MF "sys/d3d11/libgstd3d11.dll.p/gstd3d11window_swapchainpanel.cpp.obj.d" -o sys/d3d11/libgstd3d11.dll.p/gstd3d11window_swapchainpanel.cpp.obj "-c" ../gst-plugins-bad-1.24.9/sys/d3d11/gstd3d11window_swapchainpanel.cpp

  [trimmed loads of missing-braces warnings from Clang]

 ../gst-plugins-bad-1.24.9/sys/d3d11/gstd3d11window_swapchainpanel.cpp:32:10: fatal error: 'windows.ui.xaml.media.dxinterop.h' file not found
     32 | #include <windows.ui.xaml.media.dxinterop.h>
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  62 warnings and 1 error generated.
```

diff -urN gst-plugins-bad-1.24.9/gst-libs/gst/d3d11/meson.build.orig gst-plugins-bad-1.24.9/gst-libs/gst/d3d11/meson.build
--- gst-plugins-bad-1.24.9/gst-libs/gst/d3d11/meson.build.orig	2024-10-30 21:33:30.000000000 +0100
+++ gst-plugins-bad-1.24.9/gst-libs/gst/d3d11/meson.build	2024-11-20 12:49:56.619766000 +0100
@@ -77,6 +77,7 @@
 if runtimeobject_lib.found() and d3dcompiler_lib.found()
   d3d11_winapi_app = cxx.compiles('''#include <winapifamily.h>
       #include <windows.applicationmodel.core.h>
+      #include <windows.ui.xaml.media.dxinterop.h>
       #include <wrl.h>
       #include <wrl/wrappers/corewrappers.h>
       #include <d3d11_4.h>
